dev/devicelab/lib/tasks/run_tests.dart

```diff
@@ -173,12 +174,14 @@ class WindowsRunOutputTest extends DesktopRunOutputTest {
     }
   );
 
+  final String arch = Abi.current() == Abi.windowsX64 ? 'x64': 'arm64';
+
   static final RegExp _buildOutput = RegExp(
     r'Building Windows application\.\.\.\s*\d+(\.\d+)?(ms|s)',
     multiLine: true,
   );
   static final RegExp _builtOutput = RegExp(
-    r'Built build\\windows\\x64\\runner\\(Debug|Release)\\\w+\.exe( \(\d+(\.\d+)?MB\))?\.',
+    r'Built build\\windows\\.*\\runner\\(Debug|Release)\\\w+\.exe( \(\d+(\.\d+)?MB\))?\.',
```

Comment:
Can this be more specific?

```suggestion
    r'Built build\\windows\\(x64|arm64)\\runner\\(Debug|Release)\\\w+\.exe( \(\d+(\.\d+)?MB\))?\.',
```

Reply:
Thanks, I was not sure of the syntax to do this. I'll do the change.